pkg/types/clusterconfig/cluster_config.go

```diff
@@ -75,6 +75,7 @@ type CoreConfig struct {
 	ImageOperator                   string `json:"image_operator" yaml:"image_operator"`
 	ImageManager                    string `json:"image_manager" yaml:"image_manager"`
 	ImageDownloader                 string `json:"image_downloader" yaml:"image_downloader"`
+	ImageKubexit                    string `json:"image_kubexit" yaml:"image_kubexit"`
```

Comment:
There may be a few additional places where this should be added (search the code base for e.g. "istio-pilot", "istio_pilot", and "IstioPilot" to find them)

Reply:
I think those were removed

Reply:
I think it's still missing in `create.md` (but that's the only remaining one I could find)